src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar container__item calendar--month-length--31">
+        <ul class="calendar__list">
+          <li class ="calendar__day calendar__day--start-day--sat"></li>
```

Comment:
calendar__day--start-day--... you should add this class to calendar element instead calendar day element
![image](https://user-images.githubusercontent.com/54936044/223395961-8220e017-badc-40bd-93d6-6e6610165cc7.png)